compiler/optimizations/gpuTransforms.cpp

```diff
@@ -863,6 +926,11 @@ bool GpuizableLoop::extractUpperBound() {
   return true;
 }
 
+struct KernelActual {
+  Symbol* sym;
+  int8_t kind;
```

Comment:
Add a comment

`// assigned to one or more values of GpuArgKind or'd together`